lib/matplotlib/tests/test_legend.py

```diff
@@ -453,16 +453,8 @@ def test_legend_label_arg(self):
     def test_legend_label_three_args(self):
         fig, ax = plt.subplots()
         lines = ax.plot(range(10))
-        with mock.patch('matplotlib.legend.Legend') as Legend:
+        with pytest.raises(Exception):
```

Comment:
Should be possible to have an exact exception type (and matching error message)? (And below.)

Reply:
sure, done